### PR TITLE
Fixed Next-to-last page is unclickable

### PR DESCRIPTION
**Pull request by Ankit-Gupta18**
[//]: # (Thank you for uploading a PR to the Wikipedia Library!)

## Description
Problem : In review interface, the next-to-last page number is visible but is not clickable

## Rationale
[//]: # (Why is this change required? What problem does it solve?)

## Phabricator Ticket
[//]: # https://phabricator.wikimedia.org/T299867

## How Has This Been Tested?
[//]: # (- Did you add tests to your changes? Did you modify tests to accommodate your changes?)
[//]: # (- Can this change be tested manually? How?)

## Screenshots of your changes (if appropriate):
[//]: # 
prevpage

## Types of changes
What types of changes does your code introduce? Add an `x` in all the boxes that apply:
- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Minor change (fix a typo, add a translation tag, add section to README, etc.)

### CODE REVIEW

**Comment by Ankit-Gupta18** [comment]
Hi @suecarmol ,
I fixed this task T299867. But I don't know why I have got three commits here. I created a new branch T299867 and then made the changes to it and then filed a PR.
Please look into this.
Thanks and regards.

**Comment by Samwalton9** [comment]
@Ankit-Gupta18 It looks like you still have the commits from one of your old PRs here, you'll need to rework it so you only have that last commit :)

**Comment by Samwalton9** [comment]
Yep that looks better, thanks :)

File: TWLight/applications/templates/applications/application_list.html [hunk 1]
```diff
@@ -147,7 +147,7 @@ <h1 class="my-3">{{ title }}</h1>
                   </a></li>
             {% endfor %}
             <li class="page-item disabled"><a class="twl-links page-link">...</a></li>
-            <li class="page-item disabled"><a class="twl-links page-link" href="?page={{ object_list.previous_page_number}}{% if filters %}{% for filter in filters %}{% if forloop.counter0 == 0 %}&Editor={{ filter.object.pk|safe }}{% elif forloop.counter0 == 1 %}&Partner={{ filter.object.pk|safe }}{% endif %}{% endfor %}{% else %}{% endif %}">
+            <a class="twl-links page-link" href="?page={{ object_list.previous_page_number}}{% if filters %}{% for filter in filters %}{% if forloop.counter0 == 0 %}&Editor={{ filter.object.pk|safe }}{% elif forloop.counter0 == 1 %}&Partner={{ filter.object.pk|safe }}{% endif %}{% endfor %}{% else %}{% endif %}">
```
Comment (suecarmol) [inline TWLight/applications/templates/applications/application_list.html:150]: You shouldn't remove the `li` tag, just remove the `disabled` class

**Review by suecarmol — CHANGES_REQUESTED** [review]

**Comment by Ankit-Gupta18** [comment]
Hi @suecarmol ,
I have made the required changes. Please check it.

File: TWLight/applications/templates/applications/application_list.html [hunk 2]
```diff
@@ -147,7 +147,7 @@ <h1 class="my-3">{{ title }}</h1>
                   </a></li>
             {% endfor %}
             <li class="page-item disabled"><a class="twl-links page-link">...</a></li>
-            <li class="page-item disabled"><a class="twl-links page-link" href="?page={{ object_list.previous_page_number}}{% if filters %}{% for filter in filters %}{% if forloop.counter0 == 0 %}&Editor={{ filter.object.pk|safe }}{% elif forloop.counter0 == 1 %}&Partner={{ filter.object.pk|safe }}{% endif %}{% endfor %}{% else %}{% endif %}">
+            <li><a class="twl-links page-link" href="?page={{ object_list.previous_page_number}}{% if filters %}{% for filter in filters %}{% if forloop.counter0 == 0 %}&Editor={{ filter.object.pk|safe }}{% elif forloop.counter0 == 1 %}&Partner={{ filter.object.pk|safe }}{% endif %}{% endfor %}{% else %}{% endif %}">
```
Comment (suecarmol) [inline TWLight/applications/templates/applications/application_list.html:150]: Please add the class `page-item` to the `li` tag.

**Review by suecarmol — CHANGES_REQUESTED** [review]

File: TWLight/applications/templates/applications/application_list.html (hunk 2 above)
Comment (Ankit-Gupta18) [reply TWLight/applications/templates/applications/application_list.html]: > Please add the class `page-item` to the `li` tag.

Ah sorry! My bad. I have added it back again.

**Comment by Ankit-Gupta18** [comment]
Hi @suecarmol ,
I have tried to rebase my PR. Please check if its correctly done so I can go on with rebasing other PRs as well.

**Comment by Mortal303** [comment]
Hi, @suecarmol  and @Samwalton9  I want to solve this issue, could I go ahead?

**Comment by Samwalton9** [comment]
Hi, this task has been worked on, sorry

**Review by suecarmol — APPROVED** [review]
This looks good! Thank you for working on this bug fix.

**Pull request merged**